code/modules/cargo/packs/gun.dm

```diff
@@ -165,27 +165,27 @@
 /datum/supply_pack/gun/cobra20
 	name = "Cobra-20 SMG Crate"
 	desc = "Contains a .45 submachine gun, manufactured by Scaraborough Arms and chambered in .45"
-	cost = 3000
+	cost = 6000
 	contains = list(/obj/item/gun/ballistic/automatic/smg/c20r/cobra)
 	crate_name = "SMG crate"
 
 /datum/supply_pack/gun/wt550
 	name = "WT-550 Auto Rifle Crate"
 	desc = "Contains a high-powered, automatic personal defense weapon chambered in 4.6x30mm."
-	cost = 4000
+	cost = 6000
 	contains = list(/obj/item/gun/ballistic/automatic/smg/wt550)
 	crate_name = "auto rifle crate"
 
 /datum/supply_pack/gun/p16
 	name = "P16 Assault Rifle Crate"
 	desc = "Contains a high-powered, automatic rifle chambered in 5.56mm."
-	cost = 5000
+	cost = 10000
 	contains = list(/obj/item/gun/ballistic/automatic/assault/p16)
 	crate_name = "auto rifle crate"
 
 /datum/supply_pack/gun/skm
 	name = "SKM-24 Rifle Crate"
 	desc = "Contains a high-powered, automatic rifle chambered in 7.62x40mm CLIP."
-	cost = 5000
+	cost = 1000
```

Comment:
Was it intended to make the SKM fives times less expensive than originally?

Reply:
Yeah, i really think it's an u i forgot ot push the fix

Reply:
my boy